layouts/partials/fragments/hero.html

```diff
@@ -1,22 +1,22 @@
-{{ $data := ($.Scratch.Get "data") }}
-{{ "<!-- Hero -->" | safeHTML }}
-{{ $bg := $data.background | default "secondary"}}
+{{ $hero := .self.Params  }}
+{{ "<!-- hero -->" | safeHTML }}
+{{ $bg := $hero.background | default "secondary" }}
 
 <header>
-  {{ with $data.header }}
+  {{ with $hero.header }}
     <div class="jumbotron text-center header-image mb-0">
   {{ else }}
     <div class="jumbotron text-center mb-0">
   {{ end }}
-  {{ if $data.particles }}
+  {{ if $hero.particles }}
     <div id="particles-js"></div>
```

Comment:
Within this div we might need to create a few full width divs to prevent buttons jumping next to the logo, if there is enough space.